ui/src/callWithTelemetryAndErrorHandling.ts

```diff
@@ -34,7 +34,8 @@ function initContext(): [number, IActionContext] {
         },
```

Comment:
Shouldn't you be adding category to telemetry somewhere in this file?

Reply:
:-)  I think I'm going to roll it back for now anyway.  Most of the errors which I wish had categories are coming from external sources so it doesn't help in those scenarios.  Might add later.